docs/getting_started.md

```diff
@@ -60,6 +62,11 @@ In addition you will need to add a registry auth to your pull-secret to
 support deploying CI / Nightly builds. Please follow the instructions
 [here](https://mojo.redhat.com/docs/DOC-1204026) to do so.
 
+### Public key
+
+For vSphere platform, public keys are needed. Generate the public key ( if not there already )
+and place in the data directory at the root level of the project. The name of the file should be id_rsa.pub
```

Comment:
Is there any reason to use dedicated SSH keys placed to `data/` directory, instead of typically existing keys in the `~/.ssh/` directory?
Also we already have configuration for SSH key path in [`DEPLOYMENT['ssh_key']`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml#L52), wouldn't be possible to use the same key? ([There is also method for loading that key.](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/ocp.py#L68))

Reply:
as we discussed on chat we should use the libra key by default, user can change as Daniel suggested via config.